src/app/graphql.module.ts

```diff
@@ -24,6 +24,7 @@ export function createApollo(
   const auth = setContext(async operation => {
     let token: string | null = null;
     switch (operation.operationName) {
+    case 'updateOrganization':
```

Comment:
Does we really need admin rights to update our own org?

Reply:
No, we should use current customer token
![image](https://user-images.githubusercontent.com/15198683/129316041-816d7f87-896d-441a-80d9-95d17a42ffc1.png)

Reply:
In this case, we have AuthorizationError 
VirtoCommerce.ExperienceApiModule.Core.Infrastructure.Authorization.AuthorizationError: User doesn't have the required permission 'customer:update'

Reply:
Done. It temporally needs to use the xapi module from the branch VDS-2638.